README.md

```diff
@@ -61,8 +61,8 @@ details of the changes between versions.
 
 | **Version** | **Release Date** | **Source** | **Documentation** | **Download** |
 |:-------:|:------:|:-------------:|:-------:|:------------:|
-| **master** (unstable) | -- | [source](https://github.com/Unity-Technologies/ml-agents/tree/master) |  [docs](https://github.com/Unity-Technologies/ml-agents/tree/master/docs/Readme.md) | [download](https://github.com/Unity-Technologies/ml-agents/archive/master.zip) |
```

Comment:
Also, why the removal of master (unstable)?

Reply:
This is for the release branch. II do not think the release branch should point to the `moving` master/latest_release

Reply:
I think having the "master (unstable)" link should always be there.  

Will you change the 0.15.0 -> `latest_release` link before you actually create the release?

Reply:
The intent when I added this was to only link to previous releases, not master or `latest_release`